apps/docs/guide/deploying/self-hosted.md

```diff
@@ -0,0 +1,164 @@
+
+The developer can also use a service like [Fly.io](https://fly.io) to deploy the application node.
+
+We will describe below two methods of running a rollups node, the first using Docker, which can be used as the basic information to use any cloud provider, and the second using [Fly.io](https://fly.io), a managed service for running Docker containers.
+
+### Using Docker in any cloud provider
+
+Assuming the developer completed the above requirements, and has a complete `.env` file, he can run the rollups node using the following command:
+
+```shell
+$ docker run --env-file <env-file> -p 10000:10000 <image-id>
+(...)
+INFO http: listening at [::]:10000
+(...)
+INFO rollups-node: all services are ready
+(...)
+```
+
+In the examples above the `<env-file>` file is `0xc87999b8a93609268b10de25f2e49d35f80fad92813310edc585ed644a9805d3.env` and the `<image-id>` is `sha256:5c355a9bddc92aa08987f395a257a0b32a51552c969eb161386e46f9380ea2ac`.
+
+The developer is free to use any managed container solution, like Kubernetes. The rollups node is just a single container, with a single exposed port, that must be accessible from the internet.
+
+### Using fly.io
+
+[Fly.io](https://fly.io) is platform as a service that allows developers to easily deploy applications packaged as Docker containers. The rollups node can be deployed to Fly.io. Choose an `<app-name>` and follow the steps below:
+
+1. [Install the flyctl CLI](https://fly.io/docs/hands-on/install-flyctl/)
+2. [Create an account and login](https://fly.io/docs/hands-on/sign-up-sign-in/)
+3. Create an application:
+
+    ```shell
+    $ fly app create <app-name>
+    New app created: <app-name>
+    ```
+
+4. Create a postgres database application:
+
+    ```shell
+    fly postgres create --initial-cluster-size 1 --name <app-name>-database --vm-size shared-cpu-1x --volume-size 1
+    ```
+
+5. Attach database to the node application:
+
+    The previous step should have printed the connection string for the database. The developer can use it to configure the `CARTESI_POSTGRES_ENDPOINT` environment variable.
+
+    ```shell
+    fly postgres attach <app-name>-database -a <app-name>
+    ```
+
+6. Download `fly.toml` file from deployment:
+
+    ![deploy self-hosted config](./self-hosted-deployed-fly.png)
+
+7. Create secrets for sensitive configuration, replacing with the actual values:
+
+    ```shell
+    fly secrets set -a <app-name> CARTESI_BLOCKCHAIN_HTTP_ENDPOINT=<web3-provider-http-endpoint>
+    fly secrets set -a <app-name> CARTESI_BLOCKCHAIN_WS_ENDPOINT=<web3-provider-ws-endpoint>
+    fly secrets set -a <app-name> CARTESI_AUTH_MNEMONIC=<mnemonic>
+    fly secrets set -a <app-name> CARTESI_POSTGRES_ENDPOINT=<connection_string>
+    ```
```

Comment:
`fly secrets` can receive a list if you wanna make it less verbose

```suggestion
    fly secrets set -a <app-name> \
        CARTESI_BLOCKCHAIN_HTTP_ENDPOINT=<web3-provider-http-endpoint> \
        CARTESI_BLOCKCHAIN_WS_ENDPOINT=<web3-provider-ws-endpoint> \
        CARTESI_AUTH_MNEMONIC=<mnemonic> \
        CARTESI_POSTGRES_ENDPOINT=<connection_string>
```

Reply:
I any case that is not much "copy and pastable", because of the need to replace the values between `<>`